prime-router/src/main/kotlin/Translator.kt

```diff
@@ -0,0 +1,117 @@
+package gov.cdc.prime.router
+
```

Comment:
This is a new class that focuses on the process of translation. It pulls together code from the schema and metadata classes in one place.

Reply:
I'm thinking of renaming converter to serializers, which is the common Java term for these classes. I'm also thinking putting them in them in a separate folder.

Reply:
Yes to both the rename and the separate folder.